lib/string_reverse.rb

```diff
@@ -1,30 +1,27 @@
 # A method to reverse a string in place.
-def string_reverse(word)
+def string_reverse(my_string)
 
-  if word == nil
+  if my_string == nil
     return nil
   end
 
-  if word.length == 0
+  if my_string.length == 0
     return false
   end
 
-  if word.length == 1
-    return word
+  if my_string.length == 1
+    return my_string
   end
 
   i = 0
-  j = word.length - 1
+  j = my_string.length - 1
   while i <= j

```

Comment:
Pedantic: Technically, `while i < j` should be sufficient. When `i` and `j` have the same values, you don't need to swap the character with itself.